QLogger.h

```diff
@@ -205,9 +53,15 @@ class QLoggerManager
     * @param fileDest The file name and path to print logs.
     * @param module The module that will be stored in the file.
     * @param level The maximum level allowed.
+    * @param fileFolderDestination The complete folder destination.
+    * @param mode The logging mode.
+    * @param fileSuffixIfFull The filename suffix if the file is full.
     * @return Returns true if any error have been done.
     */
-   bool addDestination(const QString &fileDest, const QString &module, LogLevel level);
+   bool addDestination(const QString &fileDest, const QString &module, LogLevel level=LogLevel::Warning,
```

Comment:
Not sure about suggesting LogLevel::Warning as default. There is no default right now, but I might consider Info.

Reply:
I understand your point of view.
I set "Warning" as default value because in my own usage of QLogger, in a command line application for example, I have this behavior:

- Normal usage: LogLevel::Warning
- With "-l" option: LogLevel::Info
- With "-d" option: LogLevel::Debug
As you are the creator, the choice is yours, but I prefer "Warning" ;)

Reply:
As said, it was just preference. Since you took the time to do this modification, keep it warning! :D